### ISSUE TITLE
Adventures of Willy Beamish, The (1991)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#930]
has watermark replacementdocs
it's for the sega-cd version
according to google picture search it seems there was no further manual included than the docs that are included as extras in eXoDOS

**Comment by exoscoriae** [comment]
removed the watermark. Should we keep the manual as it is better than nothing?

**Comment by TimberlAndRE-de** [comment]
Yes, I think having a good manual from another platform is better than having none 👍🏻

**Comment by exoscoriae** [comment]
cool, sounds good to me.

**Comment by Eulisker** [comment]
I would vote for setting the reference card in the collection as the manual for Willy Beamish.

Looking through ebay i can not find evidence for any manual other than the stuff already in the collection. 
Also [here](https://ogdb.eu/index.php?section=game&gameid=3902) is a photo and text that does not mention anything else. 

Lastly there is this on the reference card:
![image](https://user-images.githubusercontent.com/81642149/140780124-67876ef8-443e-4280-ae65-ec3ce8bfe559.png)

I think thats enough for the manual.

**Comment by Eulisker** [comment]
The reference card is on the back of the Pizzarama Menu. I thought I added it to the github folder already. 
It is there now.
I would say the overall scan quality is greater in the scan we have but it is gray scale only.

You can close #1701 with this too.

**Comment by exoscoriae** [comment]
replaced

**Comment by Eulisker** [comment]
Bumping this because I still think either the pizzarama menu or the secret diary should be set as the manual. There was no classic game manual in the box. 
Having the Sega cd manual just seems wrong.

**Comment by SmilingSpectre** [comment]
Oh, and we probably doesn't need the extra "Install Guide.pdf", as it's doubled in the "Pizzarama menu.pdf".

**Comment by Dutchmagic** [comment]
My 2 cents:
For me Sierrachest.com is the golden standard for anything Sierra related. They don’t have anything beyond a manual. I have this game myself if you guys need me to look for anything I can do that as well.
